packages/nhsuk-frontend/src/nhsuk/components/breadcrumb/template.njk

```diff
@@ -1,27 +1,31 @@
+{% from "nhsuk/components/back-link/macro.njk" import backLink %}
```

Comment:
This is a lovely workaround, and keeps NHS behaviour (thank you)

**One catch:** Can you make sure the back link Sass is forwarded by the breadcrumb Sass please?

Otherwise, services that selectively import _only_ the breadcrumb Sass will have missing styles

Reply:
Done!